tests/baselines/reference/readonlyArraysAndTuples.errors.txt

```diff
@@ -52,7 +52,7 @@ tests/cases/conformance/types/tuple/readonlyArraysAndTuples.ts(25,5): error TS27
 !!! error TS2740: Type 'readonly string[]' is missing the following properties from type '[string, string]': 0, 1, pop, push, and 5 more.
         mt = rt;  // Error
         ~~
-!!! error TS2740: Type 'readonly [string, string]' is missing the following properties from type '[string, string]': pop, push, reverse, shift, and 3 more.
+!!! error TS4105: A 'readonly' tuple cannot be assigned to a mutable array-like type.

```

Comment:
Hmm, I think the fact that these lack context is a bit of a problem. If these aren't acting as elaborations, then maybe both messages could be replaced with

```
The type '{0}' is considered 'readonly' and cannot be assigned to the mutable type '{1}'.
```

Reply:
Ah yes, it slipped past me that these errors don't include  `Foo is not assignable  to Bar`. So:
  - `The type '{0}' is considered 'readonly' and cannot be assigned to the mutable type '{1}'.`

which can probably subsume both new error messages?

Reply:
Can we remove the word "considered" here?

Reply:
Works for me!